cli/.eslintrc

```diff
@@ -3,28 +3,18 @@
   "env": {
     "es6": true
   },
-  "plugins": [
-    "flowtype",
-    "prettier"
-  ],
-  "extends": [
-    "plugin:flowtype/recommended"
-  ],
-  "globals": {
-  },
+  "plugins": ["flowtype"],
+  "extends": ["plugin:flowtype/recommended", "plugin:prettier/recommended"],
```

Comment:
They [seem to recommand](https://github.com/prettier/eslint-plugin-prettier#recommended-configuration) extending the `"prettier/react"` config afterward.

Reply:
oh right on